metatron-app/metatron-discovery
# #3637 add subtotal grid

### Description

I need the subtotal function of the grid chart.

**Related Issue** : #3637 


### How Has This Been Tested?

1. Go to the chart creation screen.
2. Create a grid chart by placing at least 2 dimension values on the shelf.
3. In the Totals panel, check if the subtotal option is working properly.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
